client/gui_rpc_server_ops.cpp

```diff
@@ -664,6 +664,7 @@ static void handle_acct_mgr_info(GUI_RPC_CONN& grc) {
         || strlen(gstate.acct_mgr_info.authenticator)
     ) {
         grc.mfout.printf("   <have_credentials/>\n");
+	grc.mfout.printf("   <authenticator>%s</authenticator>\n", gstate.acct_mgr_info.authenticator);
```

Comment:
Can you replace the tab with spaces so that it properly indents?  Thanks!

Reply:
Sorry, it's my fault